substrate/bft/src/lib.rs

```diff
@@ -248,6 +248,7 @@ impl<B: Block, P: Proposer<B>> rhododendron::Context for BftInstance<B, P>
 	fn begin_round_timeout(&self, round: usize) -> Self::RoundTimeout {
 		use std::time::{Instant, Duration};
 
+		let round = round / 3;
```

Comment:
I think this is OK because we would eventually have a round that surpasses any fixed delay.